cmd/bundle/destroy.go

```diff
@@ -62,7 +63,9 @@ func newDestroyCommand() *cobra.Command {
 
 		diags = bundle.Apply(ctx, b, bundle.Seq(
 			phases.Initialize(),
-			phases.Build(),
+			mutator.ResolveVariableReferences(
+				"artifacts",
+			),
```

Comment:
I don't know if this is necessary. I was asking because there may be a subtle dependency we're missing, and I was wondering if you did the analysis on this. If it is necessary, then please include a comment to capture the nuance, otherwise we can skip this altogether.

Reply:
We should be able to skip this since `terraform destroy` depends entirely on the `id` stored in the state (IIRC). The configuration we write to `bundle.tf.json` should be irrelevant IIRC.

Reply:
If artifact variable is used in some resource, let's say in job name, the destroy will fail with the following error if variables not resolved

```
    my_job:
      name: My Job at ${artifacts.test.path}
```

```
ndrew.nester@HFW9Y94129 wheel % databricks bundle destroy -p u2m
Error: exit status 1

Error: Reference to undeclared resource

  on bundle.tf.json line 45, in resource.databricks_job.my_job:
  45:         "name": "My Job ${artifacts.test.path}",

```

Reply:
I'll add the comment